packages/components/src/Menu/Menu.tsx

```diff
@@ -7,11 +7,25 @@ import React, {
 } from "react";
 import uuid from "uuid";
 import classnames from "classnames";
+import { AnimatePresence, motion } from "framer-motion";
 import styles from "./Menu.css";
 import { Button } from "../Button";
 import { Typography } from "../Typography";
 import { Icon, IconNames } from "../Icon";
 
+const variation = {
+  overlayStartStop: { opacity: 0 },
+  startOrStop: (position: string) => {
+    let y = 10;
+
+    if (position === "below") y *= -1;
```

Comment:
why this vs `y = -10`? I feel this is more confusing

Reply:
I did this so it reverses the amount so they don't get out of sync.

Reply:
If we set the `10` as a variable(?) and then did a more explicit multiplication of the variable could that solve the out-of-sync issue while making it more apparent what's happening?

Reply:
Jesse made that comment once about magic numbers for animations in a coffeescript thing I did once

Reply:
I think that's basically what we're doing here? Starting `y` at `10` then making it negative if the position is below.